contracts/v2/ProximityScoringProxy.sol

```diff
@@ -2,14 +2,14 @@
 
 pragma solidity ^0.8.16;
 
-import {ContractStatus} from "../v1/abstract/ContractStatus.sol";
+import {ContractStatusV2} from "./abstract/ContractStatus.sol";
```

Comment:
Why do we changed ContractStatus to V2?

Reply:
Because I forgot to change dependency to HubV2 when itroducing V2 contracts, it's needed to have a proper Hub interface for interactions, but it's rather internal development things, doesn't require redeployment of all contracts where it was changed

Reply:
It just that debugger is complaining with ContractStatus v1